recon_surf/N4_bias_correct.py

```diff
@@ -334,13 +332,18 @@ def normalize_img(
     Parameters
     ----------
     itk_image : sitk.Image
+        Input image to be normalized.
     itk_mask : sitk.Image | None
+        [MISSING].
```

Comment:
Brain mask, voxels inside the mask are guaranteed to be > 0.
 "| None" is optional